op.js

```diff
@@ -70,20 +60,20 @@ const addHideSidePanelButton = function () {
 
 		// add arrow button to the panel
 		arrowHolder.appendChild(rightArrow);
-		leftPanel.insertBefore(arrowHolder, leftPanel.childNodes[0]);
+		leftPanel.insertBefore(arrowHolder, leftPanel.firstChild]);
 	}
 
 	// check out if it may be the work package view
-	if(window.location.href.indexOf("work_package") == -1) {
+	if(!~window.location.href.indexOf("work_package")) {
```

Comment:
You would have to convince me to this one.
Looks like unnecessary complication.

Reply:
`~` character is commonly used to make `indexOf` method returned 0 when nothing was found. It's vety practical in if statements. And the `!` is just a negation.

If you not convinced then I suggest to use `===` (because it's type aware) instead of `==`.

Reply:
My thinking is such that it is easy to not notice single `!` but it is hard to ignore `=== -1`.
But the change from `==` to `===` is necessary 🥇